dashboard/app/main.go

```diff
@@ -1229,48 +1237,70 @@ func makeUIJob(job *Job, jobKey *db.Key, bug *Bug, crash *Crash, build *Build) *
 	return ui
 }
 
-func fetchErrorLogs(c context.Context) ([]byte, error) {
+func fetchErrorLogs() ([]byte, error) {
+	if !appengine.IsAppEngine() {
+		return nil, nil
+	}
+
 	const (
-		minLogLevel  = 3
-		maxLines     = 100
-		maxLineLen   = 1000
-		reportPeriod = 7 * 24 * time.Hour
+		maxLines   = 100
+		maxLineLen = 1000
 	)
-	q := &log.Query{
-		StartTime:     time.Now().Add(-reportPeriod),
-		AppLogs:       true,
-		ApplyMinLevel: true,
-		MinLevel:      minLogLevel,
+
+	ctx := context.Background()
+	projId := os.Getenv("GOOGLE_CLOUD_PROJECT")
+
+	adminClient, err := logadmin.NewClient(ctx, projId)
+	if err != nil {
+		return nil, errors.New("failed to create the logging client")
```

Comment:
Add the origin error into the wrapped error, otherwise it won't be possible to understand why things are failing.

Reply:
Done.